xbmc/video/Teletext.cpp

```diff
@@ -32,11 +32,6 @@
 #include "filesystem/SpecialProtocol.h"
 #include "guilib/GraphicContext.h"
 
-#if HAVE_SDL_VERSION == 1
-#include <SDL/SDL_stdinc.h>
-#elif HAVE_SDL_VERSION == 2
-#include <SDL2/SDL_stdinc.h>
-#else
```

Comment:
no need to keep it but sdl doesn't get removed, only sdl2 will be removed by this PR